server/src/infra/repositories/album.repository.ts

```diff
@@ -189,15 +194,17 @@ export class AlbumRepository implements IAlbumRepository {
 
   @GenerateSql({ params: [{ albumId: DummyValue.UUID, assetIds: [DummyValue.UUID] }] })
   async removeAssets(asset: AlbumAssets): Promise<void> {
-    await this.dataSource
-      .createQueryBuilder()
-      .delete()
-      .from('albums_assets_assets')
-      .where({
-        albumsId: asset.albumId,
-        assetsId: In(asset.assetIds),
-      })
-      .execute();
+    for (const idChunk of _.chunk(asset.assetIds, DATABASE_PARAMETER_CHUNK_SIZE)) {
```

Comment:
Can you use `@Chunked()` here or no?

Reply:
Not with the current decorator signature, as the array is within `asset` (in `asset.assetIds`).

`AlbumAssets` type is only used in a few places, I wonder if this method and the other few ones using it should be refactored to receive `assetIds` and `albumId` separately. Then, we could pretty easily add `@Chunked()` in here too.

Reply:
That works for me.